microceph/ceph/bootstrap.go

```diff
@@ -194,36 +201,48 @@ func initMgr(s common.StateInterface, dataPath string) error {
 	return nil
 }
 
-func updateDatabase(s common.StateInterface, fsid string, adminKey string) error {
+// populateDatabase injects the bootstrap entries to the internal database.
+func populateDatabase(s common.StateInterface, fsid string, adminKey string, pubNet string) error {
 	if s.ClusterState().Database == nil {
 		return fmt.Errorf("no database")
 	}
 	err := s.ClusterState().Database.Transaction(s.ClusterState().Context, func(ctx context.Context, tx *sql.Tx) error {
 		// Record the roles.
 		_, err := database.CreateService(ctx, tx, database.Service{Member: s.ClusterState().Name(), Service: "mon"})
 		if err != nil {
-			return fmt.Errorf("Failed to record role: %w", err)
+			return fmt.Errorf("failed to record role: %w", err)
 		}
 
 		_, err = database.CreateService(ctx, tx, database.Service{Member: s.ClusterState().Name(), Service: "mgr"})
 		if err != nil {
-			return fmt.Errorf("Failed to record role: %w", err)
+			return fmt.Errorf("failed to record role: %w", err)
 		}
 
 		_, err = database.CreateService(ctx, tx, database.Service{Member: s.ClusterState().Name(), Service: "mds"})
 		if err != nil {
-			return fmt.Errorf("Failed to record role: %w", err)
+			return fmt.Errorf("failed to record role: %w", err)
 		}
 
 		// Record the configuration.
 		_, err = database.CreateConfigItem(ctx, tx, database.ConfigItem{Key: "fsid", Value: fsid})
 		if err != nil {
-			return fmt.Errorf("Failed to record fsid: %w", err)
+			return fmt.Errorf("failed to record fsid: %w", err)
 		}
 
 		_, err = database.CreateConfigItem(ctx, tx, database.ConfigItem{Key: "keyring.client.admin", Value: adminKey})
 		if err != nil {
-			return fmt.Errorf("Failed to record keyring: %w", err)
+			return fmt.Errorf("failed to record keyring: %w", err)
+		}
+
+		key := fmt.Sprintf("mon.host.%s", s.ClusterState().Name())
+		_, err = database.CreateConfigItem(ctx, tx, database.ConfigItem{Key: key, Value: s.ClusterState().Address().Hostname()})
```

Comment:
Should we check if the user passed in a `--mon-ip` and record this instead of the default microcluster ipaddr?